en-US/E.xml

```diff
@@ -89,6 +89,19 @@
 			</listitem>
 
 		</varlistentry>
+		<varlistentry id="EOL">
+		 <term>EOL</term>
+			<listitem>
+			 <para>
+				 Initialism for "end of line" (n.) and "end-of-line" (adj.)
+			 </para>
+				<para>
+				 Neither the noun nor the adjective should be capitalized unless you are documenting a GUI field, label, or similar element, in which case you should use the same capitalization. Capitalization at the beginning of a sentence is acceptable. The initialism is always uppercase.
```

Comment:
Can we rephrase this to something like "Always use uppercase for the initialism. Do not capitalize the expansion except at the beginning of a sentence. When documenting GUI objects, use the same capitalization as that shown in the GUI."

Something like that.

Reply:
Done